src/components/interactive/PreviewCard.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+
+const PreviewCard = ({
+  hoverText,
+  footerLeftInfo,
+  footerRightInfo,
+  coverContent,
+  footerLabel,
+  coverTheme,
+  ...props
+}) => (
+  <a {...props} className="fdsCard display--inlineBlock">
+    <div
+      className={cx(
+        'fdsCard-cover',
+        coverTheme && !coverContent && `fdsCard-cover--${coverTheme}`
+      )}
+    >
+      {hoverText && (
+        <div className="fdsCard-cover-hover alignChild--center--center fontStyle--caps">
+          {hoverText}
+        </div>
+      )}
+      {coverContent}
+    </div>
+    <div
+      className={cx('fdsCard-footer', {
+        'fdsCard-footer--border': !coverTheme || coverContent,
+      })}
+    >
+      <div className="fdsCard-label fontWeight--bold">{footerLabel}</div>
+      <div className="fdsCard-info">
+        <div className="fdsCard-info--right fontSize--xs color--slate">
+          {footerLeftInfo}
+        </div>
+        <div className="fdsCard-info--left fontSize--xs color--slate">
+          {footerRightInfo}
+        </div>
+      </div>
+    </div>
+  </a>
+);
+
+PreviewCard.defaultProps = {
+  footerLabel: ' ',
+  footerRightInfo: ' ',
+  footerLeftInfo: ' ',
+};
+
+PreviewCard.propTypes = {
+  footerLabel: PropTypes.string,
+  footerRightInfo: PropTypes.string,
+  footerLeftInfo: PropTypes.string,
+  hoverText: PropTypes.string,
+  coverContent: PropTypes.node,
+  coverTheme: PropTypes.oneOf(['purple', 'darkPurple', 'aqua', 'pink']),
+};
```

Comment:
Just to discuss to make sure we're on the same page:

My thoughts on JSDoc here, is to use it if it can give extra helpful context, outside of the name. So for like `footerLabel`, if the JSDoc is like "The label for the footer" then it's not necessary to give it a JSDoc. But if I can add extra helpful context, then I should add it.